webform.install

```diff
@@ -1494,13 +1494,14 @@ function webform_update_1609() {
 
   if (!db_field_exists('webform', 'back_to_form_link')) {
     $spec = array(
-      'description' => 'Boolean value of whether to display the go back to form link.',
+      'description' => 'Boolean value for whether to show or hide the "Go back to the form" link.',
       'type' => 'int',
       'size' => 'tiny',
       'not null' => TRUE,
       'default' => 1,
     );
     db_add_field('webform', 'back_to_form_link', $spec);
   }
-  return t("Webforms will now have the option to disable the display of the 'Go back to the form' link below the confirmation message. If you have overriden 'webform-confirmation.tpl.php' in your theme, you will need to manually update your theme for this setting to apply.");
+  return t('Webforms will now have the option to disable the display of the "Go back to the form" link below the confirmation message. If you have overriden <em>webform-confi
```

Comment:
Doesn't splitting a `t()` string like this cause problems?

Reply:
Oops, looks like a copy/paste problem.